packages/@aws-cdk/cloud-assembly-schema/lib/assets/schema.ts

```diff
@@ -4,7 +4,7 @@ import { FileAsset } from './file-asset';
 /**
  * Definitions for the asset manifest
  */
-export interface ManifestFile {
+export interface AssetManifest {
```

Comment:
@rix0rrr do you think we should alias this since this is a breaking change or are we all good?